portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,176 @@
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on Miami where i grew up -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://blog-www.pods.com/wp-content/uploads/2019/08/MG_6_1_Miami.jpg" alt="oh no" >
+                    <h3>Miami</h3>
+                    <p>
+                        Miami the place were I was raised. A melting pot of 
+                        different Latino cultures. Of course, We have a ton of 
+                        Cubans because you know Miami is known for their 
+                        Cubans, Mis Cubanitos. But, I love being from here and living here.  
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on bad bunny my fav artist  -->
+            <!-- maybe use the video tag and add a bad bunny video?? -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://cdn3.pitchfork.com/longform/976/badbunnyheader.jpg" alt="bad bunny" >
+                    <h3>Bad Bunny</h3>
+                    <p>
+                        Bad Bunny my all time favorite music artist.
+                        Bad Bunny is huge in the latino community and I love
+                        his music and videos. Also, I went to his last concert 
+                        in Miami. It was insane, he was soooo good live 10/10
+                        recommend. 
+                    </p>
+                </div>
+            </div>
+
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://static.iris.net.co/semana/upload/images/2018/6/1/569673_1.jpg" alt="Soccer" >
+                    <h3>Soccer(Futbol)</h3>
+                    <p>
+                        I love soccer and have loved it all my life.
+                        Growing up my family and I would watch soccer matches
+                        on the tv a lot. We alway watch parties during the Fifa World Cup
+                        to watch Colombia games. It was a lot of fun. Let's go 
+                        Colombia!
+                    </p>
+                </div>
+            </div>
+
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="images/shpe.png" alt="Shpe" >
+                    <h3>Shpe</h3>
+                    <p>
+                        Shpe is acroymn of The Society of Professional
+                        Hispanic Engineers. I have made so many friends and 
+                        Professional contact through the club. In our club we 
+                        have a saying which is "Somos Familia", we are family and
+                        we are.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+    
+    <div id="box" class="box"> 
```

Comment:
We probably don't want both id and class set in these elements.

Reply:
Just one.

Reply:
yes sorry about that I first used class but then I need to add id because in my javascript I use .getElementById so I needed to add an id but I forgot to delete the class. I generally used id if there was only one specific element I wanted to manipulate.